tools/cli/adminKafkaCommands.go

```diff
@@ -57,39 +58,69 @@ import (
 )
 
 type filterFn func(*replicator.ReplicationTask) bool
+type filterFnForVisibility func(*indexer.Message) bool
 
 const (
 	bufferSize                 = 4096
 	preambleVersion0      byte = 0x59
 	malformedMessage           = "Input was malformed"
 	chanBufferSize             = 10000
 	maxRereplicateEventID      = 999999
+
+	kafkaMessageTypeReplicationTask = 0
```

Comment:
Can we introduce a message type and make this an enum?

Reply:
Updated. Thanks